src/index.html

```diff
@@ -1,12 +1,371 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="header__logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo"
+          width="40"
+          height="40"
+        >
+      </a>
+
+      <nav class="header__navigation">
+        <ul class="header__list">
+          <li class="header__item">
+            <a 
+              href="index.html" 
+              class="header__item-link is-active"
+            > 
+              Apple 
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="index.html" class="header__item-link"> 
+              Samsung 
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="index.html" class="header__item-link">
+               Smartphones 
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a 
+              href="index.html" 
+              class="header__item-link" 
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="index.html" class="header__item-link">
+               Gadgets 
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="index.html" class="header__item-link"> 
+              Tablets 
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="index.html" class="header__item-link"> 
+              Photo 
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="index.html" class="header__item-link"> 
+              Video 
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <section class="catalog">
+        <div class="catalog__card card" data-qa="card">
+          <img
+            src="/images/imac.jpeg"
+            alt="Apple A1419 iMac 27 Retina 5K"
+            class="card__image"
+          >
+          
+          <h2 class="card__tittle">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
```

Comment:
IMHO, is a little better because we can see the content without reading the whole line
```suggestion
          <p class="card__code">
            Product code: 195434
          </p>
```